lib/storage/index_db.go

```diff
@@ -1459,10 +1460,15 @@ func (is *indexSearch) getTSDBStatusWithFiltersForDate(tfss []*TagFilters, date
 	if bytes.HasPrefix(labelNameValue, nameEqualBytes) {
 		thSeriesCountByMetricName.pushIfNonEmpty(labelNameValue[len(nameEqualBytes):], seriesCountByLabelValuePair)
 	}
+	count, err := is.getSeriesCount()
```

Comment:
It seems to be, that this series calculation is not correct.

 Need to double check it.

 What user expect to get? the number of timeseries (tsids) or the number of records at inverted index(metricIds)?